docs/manual/manual_circuit.md

```diff
@@ -1012,7 +1012,7 @@ These Pauli gadget circuits have interesting algebraic properties which are usef
 
 A Pauli gadget can be expressed as $V \, A \, V^\dagger$ where $V$ is the the circuit composed of CX gates and single qubit basis rotations on the right hand side of the Rz gate and $A$ is the Rz gate itself. This observation allows one to construct controlled Pauli gadgets much more efficently. See the [blog post](https://tket.quantinuum.com/blog/posts/controlled_gates/) on the {py:class}`~pytket.circuit.ConjugationBox` construct for more details.
 
-For further discussion see the research publication on phase gadget synthesis [^cite_cowt2020]. Ideas from this paper are implemented in TKET as the [OptimisePhaseGadgets](https://tket.quantinuum.com/api-docs/passes.html#pytket.passes.OptimisePhaseGadgets) and [PauliSimp](https://tket.quantinuum.com/api-docs/passes.html#pytket.passes.PauliSimp) optimisation passes.
+For further discussion see the research publication on phase gadget synthesis [^cite_cowt2020]. Ideas from this paper are implemented in TKET as the OptimisePhaseGadgets(inv:#.~passes.OptimisePhaseGadgets) and PauliSimp (inv:#*.PauliSimp) optimisation passes.
```

Comment:
These aren't working either as there are multiple valid import paths

Reply:
done in [c3e1312](https://github.com/CQCL/pytket-docs/pull/392/commits/c3e1312b41b9231a228c48e7b8f9a54b51267fba)